packages/client/src/runtime/core/engines/library/LibraryEngine.test.ts

```diff
@@ -110,6 +116,12 @@ test('responds to initialization error with PrismaClientInitializationError', as
       tracingHelper: disabledTracingHelper,
       env: {},
       cwd: process.cwd(),
+      inlineSchema: '',
+      inlineSchemaHash: '',
```

Comment:
Technically, given how it's implemented, at runtime `inlineSchema` and `inlineSchemaHash` would always be provided but not used?

Reply:
Yes, they are required because you can switch to the dataproxy at any time, but is only used by the `DataProxyEngine`.